src/app/onion/old-learning-object-builder/learning-object-builder.component.html

```diff
@@ -0,0 +1,63 @@
+<!-- Back Button -->
+<div class="back" routerLink="/onion/dashboard">
+  <div>
+    <i id="back-to-dashboard" class="far fa-arrow-to-left"></i>
+    Back to Dashboard
+  </div>
+</div>
+<div class="wrapper">
+  <div class="top-content" [ngClass]="learningObject.length">
+    <div class="left">
+      <div class="title-user">
+        <div class="title" [ngClass]="{'placeholder': !learningObject.name}">
+          {{ learningObject.name || 'Learning Object Name' }}
+        </div>
+        <div class="author">
+          {{ auth.name | titlecase }}
+          <span class="length {{ learningObject.length }}">
+            {{ learningObject.length }}
+          </span>
+        </div>
+      </div>
+    </div>
+    <div class="right">
+    <!--
+      <div *ngIf="!isNew" class="published-toggle">
```

Comment:
Why is this commented out?

Reply:
This is the old builder, the published toggle is removed as per Dr. Kaza's instructions a few months ago since the vernacular changed